### PR TITLE
New package: sqlcipher

### PR DESCRIPTION
**Pull request by rmsds**

### CODE REVIEW

File: var/spack/repos/builtin/packages/sqlcipher/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sqlcipher/package.py]:
```suggestion
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a minor comment, in particular for line 6 - otherwise LGTM.

File: var/spack/repos/builtin/packages/sqlcipher/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/sqlcipher/package.py]: @alalazo before we enforce this, we should make a decision on what to do about `from spack import *`. Should we remove it from existing packages? Should we remove it from the package creation template?

**Review by adamjstewart — APPROVED** [review]

File: var/spack/repos/builtin/packages/sqlcipher/package.py (hunk 1 above)
Comment (alalazo) [reply var/spack/repos/builtin/packages/sqlcipher/package.py]: Sure, what bothers me most is line 6, the `from` is definitely optional

**Comment by rmsds** [comment]
Thanks for the review, I committed the suggestion from @alalazo.

**Review by alalazo — APPROVED** [review]

**Pull request merged**